website/docs/r/storage_object_replication_policy.html.markdown

```diff
@@ -0,0 +1,117 @@
+---
+subcategory: "Storage"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_storage_object_replication_policy"
```

Comment:
do we need policy in the resource name when its really "storage object repliation"?
```suggestion
page_title: "Azure Resource Manager: azurerm_storage_object_replication"
```